sky/backends/cloud_vm_ray_backend.py

```diff
@@ -1283,57 +1286,22 @@ def _get_previously_launched_zones() -> Optional[List[clouds.Zone]]:
                     f'Cluster {cluster_name!r} (status: '
                     f'{prev_cluster_status.value}) was previously launched '
                     f'in {cloud} {region.name}. Relaunching in that region.')
-            # TODO(zhwu): The cluster being killed by cloud provider should
-            # be tested whether re-launching a cluster killed spot instance
-            # will recover the data.
             yield zones
 
-            # TODO(zhwu): update the following logics, since we have added
-            # the support for refreshing the cluster status from the cloud
-            # provider.
             # If it reaches here: the cluster status in the database gets
-            # set to INIT, since a launch request was issued but failed.
```

Comment:
Q: In master, was this comment about setting to INIT accurate? In master we also tried to stop or terminate. So it seems like it'd have a status of STOPPED or None as well?

Relatedly, were the comments below about "resetting status" -- L1306, 1327 -- accurate?